src/it/java/com/google/checkstyle/test/chapter7javadoc/rule734nonrequiredjavadoc/NonRequiredJavadocTest.java

```diff
@@ -35,4 +35,9 @@ public void testDefault() throws Exception {
         verifyWithWholeConfig(getPath("InputInvalidJavadocPosition.java"));
     }
 
+    @Test
+    public void testDefaultFormatted() throws Exception {
+        verifyWithWholeConfig(getPath("InputFormattedInvalidJavadocPosition.java"));
```

Comment:
please use String type in both Inputs to avoid removal by formatter.
![image](https://github.com/user-attachments/assets/24c0ad4e-b3ed-4256-b657-92ae137bfb4f)

What is problem with keep `// violation above ''InputInvalidJavadocPosition5' has incorrect indentation .* 2, expected .* 6.'
` ?

Extra violation from CS `''class def lcurly' has incorrect indentation level 2, expected level should be 4.'`, Looks like checkstyle problem ... but formatter removed indentation, it is formatter problem, please report issue to them.
![image](https://github.com/user-attachments/assets/2c7682a8-88cf-4892-b9dc-233b89349a37)

Reply:
please always sharer diff images like this to speedup process of review and maerge of PR.

Reply:
> please use String type in both Inputs to avoid removal by formatter.

formatter removed the import automatically so I just went with its flow

> What is problem with keep `// violation above ''InputInvalidJavadocPosition5' has incorrect indentation .* 2, expected .* 6.'`  ?

For `Top-level class` violations, these types of class names collides with each other most of the time so to prevent that I add `Extra` preceding to their names instead of changing the number at the end of name.

> but formatter removed indentation, it is formatter problem, please report issue to them.

normally no one writes code like we did in our example, it was just for testing purpose I guess it is not such a big deal to open an issue for that.

Reply:
Please import any other type instead of String. We just need any import to put comment over it.

Reply:
>Do we need an extra one?

Hmm, we loosing control over what matches to what issue.
Please add extra comment `// extra violation until https://github.com/google/google-java-format/issues/1126#issue-2440081623`

Reply:
> Please import any other type instead of String. We just need any import to put comment over it.

done

https://github.com/checkstyle/checkstyle/blob/656a6ebeab169567d4266040644c4dab1423aee0/src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule734nonrequiredjavadoc/InputInvalidJavadocPosition.java#L5-L7

https://github.com/checkstyle/checkstyle/blob/656a6ebeab169567d4266040644c4dab1423aee0/src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule734nonrequiredjavadoc/InputInvalidJavadocPosition.java#L32-L33

Reply:
> > Do we need an extra one?
> 
> Hmm, we loosing control over what matches to what issue. Please add extra comment `// extra violation until https://github.com/google/google-java-format/issues/1126#issue-2440081623`

done

https://github.com/checkstyle/checkstyle/blob/656a6ebeab169567d4266040644c4dab1423aee0/src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule734nonrequiredjavadoc/InputFormattedInvalidJavadocPosition.java#L126-L132

this extra comment is only in formatted file, not in original file